src/mavo.js

```diff
@@ -135,7 +135,7 @@ let _ = self.Mavo = $.Class(class Mavo {
 		$.bind(this.element, "mv-login.mavo", evt => {
 			if (evt.backend == (this.source || this.storage)) {
 				// If last time we rendered we got nothing, maybe now we'll have better luck?
-				if (!this.root.data && !this.unsavedChanges) {
+				if (this.inProgress !== "Loading" && !this.root.data && !this.unsavedChanges) {
```

Comment:
This is not localized?

Reply:
For now, the only thing that is localized (internally) is `Uploading` (https://github.com/search?q=repo:mavoweb/mavo%20this.inProgress&type=code), but not everywhere. However, all the in-progress events are localized and are present in `locale.en.js`. Should I submit a separate PR to localize the rest of the events? Or we can have everything done in this one.

Reply:
Btw, we already discussed in-progress events localization here: https://github.com/mavoweb/mavo/pull/799#issuecomment-983952916

And there, we have some cases where localization might be undesirable.

Reply:
One more thing: `inProgress` is an internal thing. When we show something to the user, we use the localized strings. So I believe we don't need to localize it in other places.

https://github.com/mavoweb/mavo/blob/286de53ec3276a2828f2488d1aaaa5b6170bc631/src/mavo.js#L926-L930

Reply:
We should probably lowercase the values (e.g. `"loading"` instead of `"Loading"` to make it clear this is an enum, and not user facing text), then we'd use these as localization keys for the user facing text.

Reply:
And yes, it looks like this should be a separate PR, that is merged before this one.

Reply:
We'll do it if you don't mind. :)

Reply:
Done!